### DISCUSSION TITLE
error api_id / api_hash

### DISCUSSION THREAD

**Discussion by ermanno26** [za-xap/telegram-userbot#1] [General]
hi, I'm getting a error after "config." when I change my user_id and hash.
baybe I'm missing something, but in my code I've changed "config...." with my current app_id and ash.
it's like : config.123456
or something else? thanks!

**Comment by za-xap** [comment]
So, for timechanger you can have config.py file only with:
```python
api_id = '1234567'
api_hash = '1a2blahblahblah3c4d'
``` 
With replaced id and hash to your own.

### OR

Just easier there in:
```python
client = TelegramClient("the_session", '1234567',  '1a2blahblahblah3c4d')
```
